src/NJsonSchema.Tests/Generation/AllowAdditionalPropertiesTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using NJsonSchema.Generation;
+using System;
+using System.Collections;
+using System.Linq;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Reflection;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace NJsonSchema.Tests.Generation
+{
+    public class AllowAdditionalPropertiesTests
+    {
+        public class Person
+        {
+            public string Name { get; set; }
+        }
+
+        public class Employee : Person
+        {
+            public int Salary { get; set; }
+        }
+
+#if !NET45
```

Comment:
Why no net45?

Reply:
i have copied from other tests... will remove if not needed?

Reply:
> will remove if not needed?

If it works then remove it